File: Array-Sorting-Without-Function/jaychhatrola/BubbleSort.java
package Ajava;

import java.util.Scanner;

public class BubbleSort {

	static void Sorting(int bubblesort[]) {
		int a = bubblesort.length;

		for (int i = 0; i < a - 1; i++) {
			for (int j = 0; j < a - i - 1; j++) {
				if (bubblesort[j] > bubblesort[j + 1]) {
					int temp = bubblesort[j];
					bubblesort[j] = bubblesort[j + 1];
					bubblesort[j + 1] = temp;
				}
			}

		}

		System.out.println("Sorted Array :-");
		for (int i = 0; i < a; i++) {
			System.out.print(" " + bubblesort[i]);
		}

	}

	public static void main(String[] args) {

		int n = 0;
		Scanner sc = new Scanner(System.in);

		System.out.println("Enter Array Size");
		n = sc.nextInt();

		int array[] = new int[n];

		for (int i = 0; i < n; i++) {
			array[i] = sc.nextInt();
		}

		Sorting(array);

	}

}

File: Array-Sorting-Without-Function/JayAkbari/bubblesort.c

#include <stdio.h>
#define MAXSIZE 10

void main()
{
    int array[MAXSIZE];
    int i, j, num, temp;
    printf("Enter the value of num \n");
    scanf("%d", &num);
    printf("Enter the elements one by one \n");
    for (i = 0; i < num; i++)
        {
            scanf("%d", &array[i]);
        }
    printf("Input array is \n");
    for (i = 0; i < num; i++)
        {
            printf("%d\n", array[i]);
        }
    /*   Bubble sorting begins */
    for (i = 0; i < num; i++)
        {
            for (j = 0; j < (num - i - 1); j++)
                {
                    if (array[j] > array[j + 1])
                        {
                            temp = array[j];
                            array[j] = array[j + 1];
                            array[j + 1] = temp;
                        }
                }
        }
    printf("Sorted array is...\n");
    for (i = 0; i < num; i++)
        {
            printf("%d\n", array[i]);
        }
}